src/Angor/Client/Pages/Signatures.razor

```diff
@@ -102,7 +102,19 @@
                                                     <td>@signature.TimeArrived.ToString("g")</td>
                                                     @if (signature.TimeApproved is null)
                                                     {
-                                                        <td><button class="btn btn-success" @onclick="() => ApproveSignatureCheckPassword(signature)">Approve</button></td>
+                                                        <td>
+                                                            <button class="btn btn-success" @onclick="() => ApproveSignatureCheckPassword(signature)" disabled="@signaturesRequestsApproving.ContainsKey(signature)">
```

Comment:
this way only the the approve signature button you pick will be disabled, maybe we should disable all signature while one is approving?